apps/src/p5lab/AnimationTab/AnimationListItem.jsx

```diff
@@ -260,7 +261,8 @@ const styles = {
     marginTop: 4,
     textAlign: 'center',
     userSelect: 'none',
-    overflow: 'hidden'
+    overflow: 'hidden',
+    fontSize: '13px'
```

Comment:
Why do we need to set a font size now and didn't before?

Reply:
Since we turned this into a button, it gets the default 'button' font size, which defaults to 100%. To keep this consistent with the pre-button styling, I override the button size to 13px (default body font-size).